aries-site/src/pages/templates/wizard.mdx

```diff
@@ -7,18 +7,22 @@ import {
 } from '../../examples';
 
 <Example
-  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/wizard/SimpleWizardExample.js"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/wizard/WizardValidationExample.js"
   figma="https://www.figma.com/file/GEhgYzx3b8MejBjg9gAxyA/Wizarding?node-id=1%3A2"
   screenContainer
   width="100%"
 >
-  <SimpleWizardExample />
+  <WizardValidationExample />
 </Example>
 
 ## Guidance
 
 Use a wizard when your form involves multiple steps. This makes the experience more manageable for the user and helps to eliminate error by validating inputs as the user progresses.
 
+### Header and Footer behavior
+
+The Wizard header should always be sticky. The footer, when there is sufficient height on desktop, should be placed directly below the inputs. On mobile, or when there is not sufficient height to show all inputs, the footer should be sticky on the bottom of the window.
```

Comment:
```suggestion
The Wizard header should always be sticky so that user has persistent access to previous step and cancel actions. The footer, when there is sufficient height on desktop, should be placed directly below the inputs. On mobile, or when there is not sufficient height to show all inputs, the footer should be sticky on the bottom of the window.
```